src/dndbeyond/base/utils.js

```diff
@@ -117,7 +117,7 @@ async function queryDamageTypeFromArray(name, damages, damage_types, possible_ty
 
 async function buildAttackRoll(character, attack_source, name, description, properties,
                          damages = [], damage_types = [], to_hit = null,
-                         brutal = 0, force_to_hit_only = false, force_damages_only = false, {weapon_damage_length=0}={}) {
+                         brutal = 0, force_to_hit_only = false, force_damages_only = false, {weapon_damage_length=0}={}, is_versatile) {
```

Comment:
Put the `is_versatile` inside the options object `{weapon_damage_length=0, is_versatile=false}={}` so we don't have to always pass all the arguments in order to set a specific one, which was why we added that options object.